controllers/openstackvmset_controller.go

```diff
@@ -435,7 +437,7 @@ func (r *OpenStackVMSetReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// Create controller VM objects and finally set VMSet status in etcd
 	if instance.Status.BaseImageDVReady {
 		for _, ctl := range controllerDetails {
-			err = r.vmCreateInstance(instance, envVars, &ctl, osNetBindings)
+			err = r.vmCreateInstance(instance, envVars, &ctl, osNetBindings, baseImageName)
```

Comment:
we could add the baseimage name as a new parameter to the controllerDetails ospdirectorv1beta1.Host{} then we do not need to pass a new parameter to the vmCreateInstance method.